src/EnergyPlus/UnitarySystem.cc

```diff
@@ -6812,12 +7005,15 @@ namespace UnitarySystems {
         static std::string const getUnitarySystemInput("getUnitarySystemInputData");
 
         std::string cCurrentModuleObject = "AirLoopHVAC:UnitarySystem";
+        int numUnitarySystems = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, cCurrentModuleObject);
+        int numCoilSystems = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, "CoilSystem:Cooling:DX");
+        int numAllSystemTypes = numUnitarySystems + numCoilSystems;
```

Comment:
Added to keep track of number of systems. Some may not be used but numAllSystemTypes will be equal to state.dataUnitarySystems->numUnitarySystems after all systems are read in. I am thinking this is not needed but have not tried to remove it yet.